django/contrib/admin/sites.py

```diff
@@ -314,9 +314,10 @@ def password_change_done(self, request, extra_context=None):
         request.current_app = self.name
         return password_change_done(request, **defaults)
 
-    def i18n_javascript(self, request):
+    def i18n_javascript(self, request, extra_context=None):
         """
         Displays the i18n JavaScript that the Django admin requires.
+        extra_context is not required here, but added for consistency with rest of admin views.
```

Comment:
Please wrap this line to 79 chars or fewer.

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style